cypress/e2e/unit_tests/uninstall.spec.ts

```diff
@@ -17,8 +17,15 @@ describe('Epinio uninstallation testing', () => {
     cy.epinioUninstall();
   });
 
-  it('Remove the Epinio helm repo', () => {
-    cy.removeHelmRepo();
-  });
-
+  if (Cypress.env('experimental_chart_branch') != null) {
```

Comment:
We use just one `it('Install Epinio')` block so why we have two for uninstall? Please unify.